broadcast/index.md

```diff
@@ -20,8 +20,16 @@ interesting developments in the ecosystem around Trino.
 ## Upcoming episodes
 
 <dl>
-<dt>Dec 2024: Trino Community Broadcast 68</dt>
-<dd>Topic to be determined - contact us if you are interested to join as a guest</dd>
+<dt>17 Jan 2024: Trino Community Broadcast 68 - Year of the snake</dt>
```

Comment:
I need to stop using the wrong GitHub account. 😆 
```suggestion
<dt>16 Jan 2024: Trino Community Broadcast 68 - Year of the Snake</dt>
```

Reply:
Thanks for catching this..